libcontainer/utils/utils_unix.go

```diff
@@ -98,3 +107,74 @@ func NewSockPair(name string) (parent *os.File, child *os.File, err error) {
 	}
 	return os.NewFile(uintptr(fds[1]), name+"-p"), os.NewFile(uintptr(fds[0]), name+"-c"), nil
 }
+
+type schedAttr struct {
```

Comment:
I would want to be very sure that the padding is handled here exactly the same as with C. Can we make this a CGo structure? Or does Go have padding guarantees that match C?